githooks/pre-commit

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
+#
+# A pre-commit Git hook.
+#
+# Currently checks the formatting of all Rust files staged for commit,
+# failing if any need to be reformatted using `rustfmt`.  Note that the
+# hook does not modify any file.
+#
+# Authors
+#
+# The Veracruz Development Team.
+#
+# Copyright and licensing
+#
+# See the `LICENSE_MIT.markdown` file in the Veracruz repository root
+# directory for copyright and licensing information.
+
+HAS_ISSUES=0
+FIRST_FILE=1
+
+echo "Checking code formatting of files staged for commit."
+
+for file in $(git diff --name-only --staged *.rs); do
+    RUSTFMT="$(rustfmt --check --skip-children --unstable-features $file)"
+    if [ "$RUSTFMT" != "" ]; then
+        if [ $FIRST_FILE -eq 0 ]; then
+            echo -n ", "
```

Comment:
That comma comes out at the start of a line and breaks the column. I don't think commas are needed if each file has its own line.

Reply:
This has been removed.